lookout/style/format/code_generator.py

```diff
@@ -67,14 +68,19 @@ def apply_predicted_y(self, vnodes: Sequence[VirtualNode],
         :param vnodes_y: Sequence of the labeled `VirtualNode`-s corresponding to labeled samples.\
                          Should be ordered by start position value.
         :param y_pred: The model predictions for `vnodes_y` `VirtualNode`-s.
+        :param rule_winners: Sequence of applied rules to generate y_pred.
+        :param rules: Rules that were used for prediction.
         :return: The list of VirtualNode-s with adjusted labels. "y_old" attribute of each node \
                  contains the previous label.
         """
         result = []
-        for vnode, y_new in self._iterate_vnodes(vnodes, vnodes_y, y_pred):
+        for vnode, y_new, applied_rule in \
+                self._iterate_vnodes(vnodes, vnodes_y, rule_winners, rules):
             vnode = vnode.copy()
             if y_new != vnode.y:
                 vnode.y, vnode.y_old = y_new, vnode.y
```

Comment:
As Vadim asked I extract `y_new` from `rules.rules[rule_winners[i]].stats.cls` but I keep triplet `vnode.y, vnode.y_old, vnode.applied_rule`. However, it is possible to drop `vnode.y_old` and keep original target value in `vnode.y`. But it requires additional work. 
It is up to reviewers, I can do that. I do not feel a big need for this change because we want to add Hugo's annotation thing so all this stuff will be gone.